shared/fs/common/path-item-action-container.js

```diff
@@ -74,9 +53,10 @@ const mapDispatchToProps = (dispatch: Dispatch) => ({
     : {}),
 })
 
-const getRootMenuItems = (stateProps, dispatchProps) => {
-  const {path, pathItem, fileUIEnabled, _username} = stateProps
+const getRootMenuItems = (stateProps, dispatchProps, path: Types.Path) => {
+  const {_pathItems, fileUIEnabled, _username} = stateProps
   const {_showInFileUI, _saveMedia, _shareNative, _download, _ignoreFolder} = dispatchProps
+  const pathItem = _pathItems.get(path) || Constants.makeUnknownPathItem()
```

Comment:
you can use .get(path, Constants.makeUnknownPathItem()) or even bake that into a helper in constants

Reply:
I've been using `.get(path, Constants.makeUnknownPathItem())` but forgot to fix this one when copy/pasting.

I'm curious why a helper in constants would be better in this case? I assume it's gonna look like `getPathItem(pathItems: Map<...>, path: Path)` which seems similar to this form.